types/postcss-prefix-selector/tsconfig.json

```diff
@@ -15,7 +15,7 @@
         "types": [],
         "noEmit": true,
         "forceConsistentCasingInFileNames": true,
-        "esModuleInterop": true
+        "exactOptionalPropertyTypes": true
```

Comment:
please leave config with defaults, we code for 'exactOptionalPropertyTypes' support, but without enforcing it via hardwired property.
`esModuleInterop` options should not be here either.